packages/editor/src/store/test/selectors.js

```diff
@@ -1334,20 +1382,21 @@ describe( 'selectors', () => {
 				saving: {
 					requesting: true,
 				},
+				getCurrentUser() {},
```

Comment:
Why those functions are part of the state?

Reply:
It's part of the registry selector mocking done above. These parts don't have a legacy editor store counterpart like `saving` does. So, I just inlined a mock getter instead of making up a piece of state which could be confusing. This sort of "remembers" that difference.